docs/adding-a-resource.md

```diff
@@ -0,0 +1,119 @@
+# Adding a new resource
+
+All resources are written using Markdown. Markdown files are stored in the
+`content` folder. This is where you'll find or create new `.md` files for
+resources.
+
+Useful links:
+
+- [Example resource file](docs/example-resource.md)
+
+- [Example PR](https://github.com/DFE-Digital/improve-school-workload-and-wellbeing/pull/39)
```

Comment:
I think some people might not realise why an example PR is useful.

Maybe:
This is an example pull request. This is how changes happen. If you copy the changes in this pull request, you can add a new resource. (or something less lame)